release-1.5/release-notes-draft.md

```diff
@@ -5,12 +5,12 @@
 ## Major Themes
 
 - Improved Federation Support
+  - New command: `kubefed`
```

Comment:
I will call out that the objective of the new CLI is to make it easier to deploy federation control plane.